lib/file_system/backends/fs_inotify.ex

```diff
@@ -63,7 +63,7 @@ defmodule FileSystem.Backends.FSInotify do
   end
 
   defp executable_path(:system_path) do
-    System.find_executable("inotifywait")
+    System.find_executable("/bin/sh")
```

Comment:
I think we should let `executable_path/1` return the path of `inotifywait` here, and make the major [command](https://github.com/falood/file_system/blob/master/lib/file_system/backends/fs_inotify.ex#L110) to run `/bin/sh`.